<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="vinita-HP-Pavilion-Notebook" failures="1" tests="3" name="api testing" time="13.435" errors="0" timestamp="01 Sep 2019 07:09:08 GMT">
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="feature" time="7.966"/>
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="feature" time="0.249"/>
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="feature" time="4.466">
    <failure type="cucumber.runtime.CucumberException" message="java.lang.AssertionError: Status check FAILED expected [200] but found [404]">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: Status check FAILED expected [200] but found [404]
at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
Caused by: java.lang.AssertionError: Status check FAILED expected [200] but found [404]
at reqres.helpers.HelperTestMethods.checkStatusIs200(HelperTestMethods.java:15)
at reqres.e2e.steps.user.ValidateUser.the_user_should_be_validated(ValidateUser.java:90)
at âœ½.Then the user should be validated(user/ValidateUser.feature:20)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- api testing -->
